fuzzers/056-pip-rem/Makefile

```diff
@@ -1,6 +1,6 @@
 MAKETODO_FLAGS=--re ".*" --not-endswith ".VCC_WIRE"
 GENERATE_FLAGS=--todo ../todo.txt
-N = 48
+N = 24
```

Comment:
Is `24` enough for each iteration to get enough information per each iteration? (For kintex7 and zynq7 as well)

Reply:
I was testing on kintex7.  24 should be more than enough actually.